README.md

```diff
@@ -459,6 +464,18 @@ succeed. If your users are coming in over the Internet (and not over LAN), this
 The public IP address is discovered via [STUN]. STUN servers can be specified with the ``JVB_STUN_SERVERS``
 option.
 
+### Error Tracking
+To stay informed about problems occuring on your deployment you can use [Sentry](https://sentry.io/).
+
+To activate and configure Sentry you have to set the `SENTRY_DSN` environment variable for the containers you want to monitor. Each container should have it's own DSN. More information including all configartion options you can find [here](https://docs.sentry.io/clients/java/config).
+
+You can also set a release and environment to filter for possible errors and warnings more easily. These can be configured by defining `SENTRY_ENVIRONMENT` and `SENTRY_RELEASE` environment variables.
+
+Currently available for:
+* JVB
+* Jicofo
+* Jigasi
+
```

Comment:
These change need to be applied to https://github.com/jitsi/handbook/blob/master/docs/devops-guide/docker.md (cf: #601)

Reply:
https://github.com/jitsi/handbook/pull/18